src/dotnet-interactive-vscode/src/tests/unit/acquisition.test.ts

```diff
@@ -0,0 +1,333 @@
+
+    function getInteractiveVersionThatReturnsNoVersionFound(dotnetPath: string, globalStoragePath: string): Promise<string | undefined> {
+        return new Promise<string | undefined>((resolve, reject) => {
+            resolve(undefined);
+        });
+    }
+
+    function getInteractiveVersionThatReturnsSpecificValue(version: string): { (dotnetPath: string, globalStoragePath: string): Promise<string | undefined> } {
+        return function(dotnetPath: string, globalStoragePath: string): Promise<string | undefined> {
+            return new Promise<string | undefined>((resolve, reject) => {
+                resolve(version);
+            });
+        };
+    }
+
+    function createEmptyToolManifest(dotnetPath: string, globalStoragePath: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            const manifestDir = path.join(globalStoragePath, '.config');
+            fs.mkdirSync(manifestDir);
+            const manifestPath = path.join(manifestDir, 'dotnet-tools.json');
+            const manfiestContent = {
+                version: 1,
+                isRoot: true,
+                tools: {}
+            };
+            fs.writeFile(manifestPath, JSON.stringify(manfiestContent), () => resolve());
+        });
+    }
+
+    function createToolManifestThatThrows(dotnetPath: string, globalStoragePath: string): Promise<void> {
+        throw new Error('This function should have never been called.');
+    }
+
+    function installInteractiveTool(args: InstallInteractiveArgs, globalStoragePath: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            const manifestPath = path.join(globalStoragePath, '.config', 'dotnet-tools.json');
+            fs.readFile(manifestPath, (err, data) => {
+                let manifestContent = JSON.parse(data.toString());
+                manifestContent.tools['microsoft.dotnet-interactive'] = {
+                    version: args!.toolVersion,
+                    commands: [
+                        'dotnet-interactive'
+                    ]
+                };
+                fs.writeFile(manifestPath, JSON.stringify(manifestContent), () => resolve());
+            });
+        });
+    }
+
+    function installInteractiveToolWithSpecificVersion(version: string): { (args: InstallInteractiveArgs, globalStoragePath: string): Promise<void> } {
+        const overrideArgs = {
+            dotnetPath: 'dotnet',
+            toolVersion: version
+        };
+        return (args: InstallInteractiveArgs, globalStoragePath: string) => {
+            return installInteractiveTool(overrideArgs, globalStoragePath);
+        };
+    }
+
+    function installInteractiveToolThatAlwaysThrows(args: InstallInteractiveArgs, globalStoragePath: string): Promise<void> {
+        throw new Error('This function should have never been called.');
+    }
+
+    function report() {
+        // noop
+    }
+
+    it("global storage doesn't exist", async () => {
```

Comment:
The name expresses just a precondition. It would be more helpful if it expressed an expected outcome, like `it("throws existential panic when nothing exists"`.

I think the expectation in this case is it uses a default/pre-installed version?